pandas/core/groupby/groupby.py

```diff
@@ -1435,6 +1446,7 @@ def result_to_bool(result: np.ndarray, inference: type) -> np.ndarray:
             post_processing=result_to_bool,
             val_test=val_test,
             skipna=skipna,
+            masked=False,
```

Comment:
Had originally used `masked` being included in kwargs as a signal for whether to pass `masked` into the groupby func. But the original value here wasn't relevant since it gets set before being used.

This is definitely confusing and a poor solution, changed to instead use a positional arg

Reply:
Thanks, the update looks clearer to follow now